simms_tests/predict_from_cat_tests.py

```diff
@@ -76,7 +76,8 @@ def test_compute_vis_stokes_I_only(self):
         sources = [self.source]
         
         ncorr = 2
-        vis = compute_vis(sources, self.uvw, self.freqs, ncorr, False, 'linear', ra0=self.ra0, dec0=self.dec0)
+
+        vis = compute_vis(sources=sources, uvw=self.uvw, freqs=self.freqs, ncorr=ncorr, polarisation=False, pol_basis='linear', ra0=self.ra0, dec0=self.dec0)
```

Comment:
Very good change, especially for the polarisation option. The floating 'False' makes it hard to read the code.